c2corg_api/models/waypoint.py

```diff
@@ -139,7 +139,7 @@ class _WaypointMixin(object):
     phone_custodian = Column(String(50))
 
     # gardiennage (gite, camp_site, hut, abri, bivouac, base_camp)
-    custodianship = Column(Boolean)
+    custodianship = Column(enums.custodianship_type)
```

Comment:
Ok. No, you don't have to change the commit message for me. I just wanted to make sure that we are talking about the same thing.